tests/unit/test_sysctl.py

```diff
@@ -208,8 +208,9 @@ def test_create_snapshot(self, mock_load, mock_output):
         config._desired_config = {"vm.swappiness": "0", "other_value": "10"}
         snapshot = config._create_snapshot()
 
-        assert mock_output.called_with(["sysctl", "vm.swappiness", "-n"])
-        assert mock_output.called_with(["sysctl", "other_value", "-n"])
+        mock_output.assert_called_once_with(
+            ["sysctl", "-n", "vm.swappiness", "other_value"], stderr=-2, universal_newlines=True
+        )
```

Comment:
The old test was just bad, wasn't it?
Meaning that the 2 settings were fetched serially before, and in a single command now. The code change must've been done without correcting the tests at some point prior, mustn't it?

I wonder if `stderr=-2` is actually significant, but then while the value could be checked against `ANY`, the presence of the kwarg cannot be easily ignored, which is a bit of an impasse.

Happy to have this change regardless :)